cms/static/cms/js/spec/main.js

```diff
@@ -200,10 +201,19 @@
                 deps: ['jquery']
             },
             'cms/js/main': {
-                deps: ['coffee/src/ajax_prefix']
+                deps: ['coffee/src/ajax_prefix', 'js/src/logger']
             },
             'coffee/src/ajax_prefix': {
                 deps: ['jquery']
+            },
+            'js/src/logger': {
+                exports: 'Logger',
+                init: function() {
+                    return {
+                        log: function() {}
```

Comment:
Nit: I'd add a comment in here saying that you are stubbing out logging just for the tests.